### DISCUSSION TITLE
Cannot create nested collections on 1.32.0

### DISCUSSION THREAD

**Discussion by AdiTDR** [dani-garcia/vaultwarden#4852] [Q&A] [answered]
### Subject of the issue
Cannot create nested collections on 1.32.0

### Deployment environment

* Vaultwarden version: v1.32.0
* Web-vault version: v2024.6.2b
* OS/Arch: linux/x86_64
* Running within a container: true (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: false (X-Forwarded-For)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: n/a
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.46.0
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Steps to reproduce
Access organization vault, click on New - Collection. 

### Expected behaviour
Field **Nest collection under** should be populated with all other already created collection names.

### Actual behaviour
Field **Nest collection under** is populated only with **No collection** value.

### Troubleshooting data

![image](https://github.com/user-attachments/assets/de342aea-8aec-430c-961f-32ae6f0ae78f)

**Comment by BlackDex** [comment, marked as answer]
Run this via the admin portal then it should work.

**Reply by tessus** [reply]
It seems to me that bw made the UI less intuitive and removed functionality that was easily accessible in previous versions.
Their API is also slightly inconsistent. TBH, I'm rather puzzled by all this, especially since bw grew into a company with more than 200 employees.

**Reply by BlackDex** [reply]
It might be a simple as a bug or oversight in this version and it's resolved in a newer version already. I have not yet compared them my self. But we are currently fixed to 2024.6.2 for now until we created all the new collection and roles into Vaultwarden.

**Comment by AdiTDR** [comment]
> Run this via the admin portal then it should work.

Indeed, nested collection creation works from the admin console. Thanks for the workaround!
